clients/zbctl/cmd/root.go

```diff
@@ -58,9 +62,13 @@ func Execute() {
 }
 
 func init() {
-	rootCmd.PersistentFlags().StringVar(&addressFlag, "address", "", "Specify the Zeebe addressFlag")
+	rootCmd.PersistentFlags().StringVar(&addressFlag, "address", "", "Specify a contact point address")
 	rootCmd.PersistentFlags().StringVar(&caCertPathFlag, "certPath", "", "Specify a path to a certificate with which to validate gateway requests")
-	rootCmd.PersistentFlags().StringVar(&credsPathFlag, "credsPath", "", "Specify a path to a Zeebe credentials file")
+	rootCmd.PersistentFlags().BoolVar(&useOAuthFlag, "useOAuth", false, "Specify if an OAuth access token should be used")
+	rootCmd.PersistentFlags().StringVar(&clientIdFlag, "clientId", "", "Specify a client identifier to request an access token")
```

Comment:
Should we specify here as well that the environment variables are supported? It might not be obvious to users.

Reply:
I didn't because it wasn't documented here for the address. I thought about just updating the docs after

Reply:
We should update the docs for sure, but maybe also document them here, since if you're a `zbctl` user it's just much more common/ergonomic to type `zbctl --help` when you're unsure of something.